### DISCUSSION TITLE
How to plot in-grain misorientation axis (IGMA)

### DISCUSSION THREAD

**Discussion by jucego333** [mtex-toolbox/mtex#1426] [Ask Anything] [answered]
Dear users of MTEX

 I am interested to plot in-grain misorientation axis (IGMA) for a given grain (please see the picture state below)
<img width="454" alt="00001" src="https://user-images.githubusercontent.com/78469467/167349492-93602af7-f00a-4b59-85b8-62171cd19890.png">
Is it possible to do this MTEX? 
I know that is possible to select grain in MTEX
(https://mtex-toolbox.github.io/SelectingGrains.html) However, I am not totally sure how to plot the IGMA into an IPF. 
I will deeply appreciate any guidelines concerning this. Thanks

**Comment by kilir** [comment, marked as answer]
Hi,
as far as I can see those are misorientation axes between all neighboring points (e.g. in a grain), right? Have a look here https://github.com/mtex-toolbox/mtex/discussions/1370.
To plot misorientation axes in crystal coordinates, see https://mtex-toolbox.github.io/SubGrainBoundaries.html#6.

I think this would not make any sense for noisy data. Maybe deciding for a lower threshold and weighting by misorientation angle could yield more telling results.

Cheers,
Rüdiger

**Reply by jucego333** [reply]
Thank you so much. I will consider it this
